configs/ext/README.md

```diff
@@ -0,0 +1,20 @@
+Summary
+-------
+
+All files in this directory are distributed under the normal Mbed TLS dual Apache 2.0 or GPLv2-or-later
+license, and are copyright The Mbed TLS Contributors.
+
+Background
+-----------
```

Comment:
(trivial) number of `-`s doesn't seem to match the number of characters in `Background` :)